packages/trader/src/App/Components/Form/TimePicker/dialog.jsx

```diff
@@ -118,11 +118,11 @@ const Dialog = ({ preClass, selected_time, end_times, start_times, onChange, cla
 
 Dialog.propTypes = {
     className: PropTypes.string,
-    end_time: PropTypes.oneOfType([PropTypes.number, PropTypes.string, PropTypes.object]),
+    end_times: PropTypes.oneOfType([PropTypes.number, PropTypes.string, PropTypes.object]),
     onChange: PropTypes.func,
     preClass: PropTypes.string,
     selected_time: PropTypes.oneOfType([PropTypes.number, PropTypes.string, PropTypes.object]),
-    start_time: PropTypes.oneOfType([PropTypes.number, PropTypes.string, PropTypes.object]),
+    start_times: PropTypes.oneOfType([PropTypes.number, PropTypes.string, PropTypes.object]),
```

Comment:
is there any possibility to use just one type for these?
string, number and object,they are somehow against solid principles